lib/cdo/secrets_config.rb

```diff
@@ -70,9 +151,13 @@ def to_s
       @table.select {|_k, v| v.to_s.match(SECRET_REGEX)}.each do |key, value|
         cdo_secrets.required(*value.to_s.scan(SECRET_REGEX).flatten)
         @table[key] = Cdo.lazy do
-          value.is_a?(Secret) ?
-            cdo_secrets.get!(value.key) :
+          if value.is_a?(Secret)
+            value.lookup(cdo_secrets)
+          else
+            # TODO: Do we need to modify this use case as well to get a stack specific secret?
```

Comment:
does this still need to be addressed, or are we planning to leave the log in for a bit to confirm that it isn't being used?